### ISSUE TITLE
ReferenceError: Popper is not defined

### ISSUE THREAD

**Issue by nicholasinclan** [twbs/bootstrap#25040]
Error message: **ReferenceError: Popper is not defined**
![poppererror](https://user-images.githubusercontent.com/7190160/34175426-6297f8f6-e4ca-11e7-8b26-a4123ceecdb3.png)

- bootstrap: v4.0.0-beta.2
- wordpress: 4.9.1
- browser: firefox quantum 57.0.2 (64-bit)
- site: staging.church360.co

what I've tried (screen shot is of attempt #2)
1. creating custom popper file: https://unpkg.com/popper.js@1.12.9/dist/umd/popper.js
2. loading bootstrap.bundle.min.js
3. source link to: https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.13.0/umd/popper.min.js

**Comment by Johann-S** [comment]
Sorry, but we cannot help with how-to or general troubleshooting issues here; only bug reports and feature requests. You'll want to ask around in the official Slack team or Stack Overflow. [See our readme for more details](https://github.com/twbs/bootstrap#community).

BTW something like our [starter template](https://getbootstrap.com/docs/4.0/getting-started/introduction/#starter-template) will work perfectly : 
```html
<!doctype html>
<html lang="en">
  <head>
    <title>Hello, world!</title>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <!-- Bootstrap CSS -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
  </head>
  <body>
    <h1>Hello, world!</h1>

    <!-- Optional JavaScript -->
    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
  </body>
</html>

**Comment by nicholasinclan** [comment]
Johann,
thanks for the response. 

your suggestion worked for me with a bit of tweaking

In case anyone else comes across this, I placed the below in my wordpress functions.php during wp_enqueue_scripts. a bit unique way to load scripts but all errors were resolved. 

`	echo "<script src='https://code.jquery.com/jquery-3.2.1.slim.min.js'></script>";
	echo "<script src='https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js'></script>";
	echo "<script src='https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js'></script>";`
